gslab_scons/builders/build_matlab.py

```diff
@@ -53,10 +54,8 @@ def build_matlab(target, source, env):
     # Matlab options
     if misc.is_unix():
         options = '-nosplash -nodesktop'
-    elif sys.platform == 'win32':
-        options = '-nosplash -minimize -wait'
     else:
-        raise Exception("Unknown OS")
+        options = '-nosplash -minimize -wait'
 
     # Run matlab on source file
     shutil.copy(source_file, 'source.m')
```

Comment:
@arosenbe - I don't know offhand why we copy the script, and I think it would be cleaner not to do so if making a change makes neither running nor logging more complicated. I'll look into this and report back.